modules/lti/src/main/resources/tools/series/series.js

```diff
@@ -115,6 +121,32 @@ function loadPage(page) {
         }
       }
 
+      // Download buttons
+      // Create a button for each track that contains a link to a mp4 or webm
+      var buttonData = [];
+      var tracks = episode.mediapackage.media.track;
+      tracks = Array.isArray(tracks) ? tracks : [tracks];
+      for (var k = 0; k < tracks.length; k++)
+      {
+        if( tracks[k].url.endsWith('mp4') || tracks[0].url.endsWith('webm')) {
+
+          // Calculate filesize
+          var fileSize = (tracks[k].video.bitrate / 8 / 1000 / 1000)      // Converts to byte, kilobyte, megabyte /s
+            * (tracks[k].video.framecount / tracks[k].video.framerate);
+
+          // Collect output to mustache
+          buttonData.push ({
+            'download-button-name' : capitalize( tracks[k].type.split('/')[0] )
+            + '<br /> ('
+            + tracks[k].video.resolution
+            + ', '
+            + fileSize.toFixed(2) + 'MB'
+            + ')',
```

Comment:
I don't think this is how it should be done. Rather than adding HTML in this string and then using `{{{ x }}}` to not escape the string, you should use two strings (`download-button-name` and `download-description` or something like that). Using `{{{ }}}` in Mustache should be avoided if possible.